src/scores/continuous/interval_impl.py

```diff
@@ -0,0 +1,100 @@
+"""
+Implementation of quantile interval score
+"""
+from typing import Optional
+
+import xarray as xr
+
+from scores.functions import apply_weights
+from scores.typing import FlexibleDimensionTypes
+from scores.utils import check_dims, gather_dimensions
+
+
+def quantile_interval_score(  # pylint: disable=R0914
+    fcst_lower_qtile: xr.DataArray,
+    fcst_upper_qtile: xr.DataArray,
```

Comment:
There may be some confusion on my part because in the referenced document "Corollary 5.2" immediately preceeds section "5.2 Interval Score". The interval score section seems to more closely describe what's going on in the code. Can you please check against the reference and help clarify the documentation about what's being used.

Reply:
You are right that in the reference article, the function is more general considering arbitrary number of quantiles, but this function we have here is just for two quantile levels as it is most common use of this score. But if there is any use case for having more than two quantile levels, we can later add a function as extension to the existing one.